packages/block-library/src/legacy-widget/edit/index.js

```diff
@@ -47,6 +50,10 @@ class LegacyWidgetEdit extends Component {
 			hasPermissionsToManageWidgets,
 			setAttributes,
 		} = this.props;
+		const visibleLegacyWidgets = pickBy(
```

Comment:
Do we expect `availableLegacyWidgets` to change after page load? If not then we'd get a very minor performance improvement by moving this `pickBy` outside of the component's render function.

Reply:
I refactor the code and used a dedicated component for the placeholder where we can easily use the memo hook so visibleLegacyWidgets is only computed one time if availableLegacyWidgets never changes, if it changes visibleLegacyWidgets are updated :)